docker/files/usr/local/bin/auth_init

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
```

Comment:
Oh, that's a good idea! I did not implemented it at first, as it was part of the scenario to create a user, but lets' keep it :wink:

Reply:
Thanks)

Reply:
P.S. I did not implemented it either, it's copy pasted from openedx-docker)